src/nni_manager/core/test/mockedTrainingService.ts

```diff
@@ -102,7 +102,7 @@ class MockedTrainingService extends TrainingService {
         return false;
     }
 
-    public cancelTrialJob(trialJobId: string): Promise<void> {
+    public cancelTrialJob(trialJobId: string, byAssessor: boolean = false): Promise<void> {
```

Comment:
I suggest to rename byAssessor to isEearlyStopped

Reply:
sure